### ISSUE TITLE
Compendium Overkill Monster Attacks are missing

### ISSUE THREAD

**Issue by CatbotTsumi** [Skyl3lazer/giffyglyph-monster-maker-continued#40]
Compendium Overkill Monster Attacks are missing.
![image](https://github.com/Skyl3lazer/giffyglyph-monster-maker-continued/assets/63572588/9d38e85f-7a48-40f9-8c00-c79728622a75)

**Comment by Skyl3lazer** [comment]
Yeah I've added the folder but overkill attack examples haven't yet been implemented. They're some of the additional examples mentioned in the 1.0 notes.
